contents/docs/deploy.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Deploy PostHog
+sidebar: Docs
+---
+
+PostHog offers a number of different deployment options:
+
+-   [PostHog Cloud](#posthog-cloud) - Create a new PostHog Cloud deployment with just a few clicks.
+    -   Recommended as it's **the fastest way** to try PostHog out.
+    -   Comes with **extra features for reliability and security** and automatic updates.
+-   [Hobby](#hobby-deployment) - Recommended for testing or very small hobby projects _(&lt;100k events/month)_
+-   [PostHog Self-hosted](#posthog-self-hosted) - Recommended for most self-host deployments.
+
+## PostHog Cloud
+
+[PostHog Cloud](/pricing) is our managed hosting service and let's you start using PostHog in only a few clicks.
+We generally recommend this option for most users as it's the easiest to try out and offers the most features out of the box:
+
+-   **1 million events included _free_ every month!**
+-   Highly-scalable infrastructure to handle large event volumes _(>100m events/month)_
+-   Automatic updates when new features are released
+-   Unlimited teammates and SSO/SAML _(Enterprise only)_
+
+<div class="flex flex-col items-stretch lg:flex-row lg:items-center justify-between bg-gray-accent-light py-4 px-6 rounded-sm mb-4 space-y-3 lg:space-y-0 lg:space-x-3">
```

Comment:
<img width="682" alt="Screenshot 2022-07-21 at 11 24 05" src="https://user-images.githubusercontent.com/84011561/180192053-7854fc57-9d93-4881-9872-743cf5963221.png">

This doesn't display well in the Docs' dark mode option.